kautomator/kautomator-framework/src/main/kotlin/com/kaspersky/components/kautomator/component/switch/UiSwitch.kt

```diff
@@ -5,14 +5,15 @@ import com.kaspersky.components.kautomator.component.common.assertions.UiBaseAss
 import com.kaspersky.components.kautomator.component.common.builders.UiViewBuilder
 import com.kaspersky.components.kautomator.component.common.builders.UiViewSelector
 import com.kaspersky.components.kautomator.component.common.views.UiBaseView
+import com.kaspersky.components.kautomator.component.text.UiTextViewAssertions
 
 /**
  * Ui View with [UiSwitchableActions] and [UiBaseAssertions]
  *
  * @see UiSwitchableActions
  * @see UiBaseAssertions
  */
-class UiSwitch : UiBaseView<UiSwitch>, UiSwitchableActions, UiBaseAssertions {
+class UiSwitch : UiBaseView<UiSwitch>, UiSwitchableActions, UiTextViewAssertions {
```

Comment:
Are you sure all the method provided by `UiTextViewAssertions` work with `UiSwitch`?

Reply:
I think, yes.
![image](https://user-images.githubusercontent.com/2050571/81633965-8fbde680-9416-11ea-8004-bd92d78339b2.png)